.zenodo.json

```diff
@@ -0,0 +1,35 @@
+{
+  "creators": [
+    {
+      "affiliation": "Earth Lab, University of Colorado - Boulder",
+      "name": "Wasser, Leah",
+      "orcid": "0000-0002-8177-6550"
+    },
+    {
+      "affiliation": "Biological Informatics Centre of Excellence Agriculture and Agri-food Canada",
+      "name": "Cranston, Karen",
+      "orcid": "0000-0002-4798-9499"
+    },
+  ],
+  "contributors": [
```

Comment:
@nkorinek can you please list time as a creator as well? he did originally create this package so i think it's important to list him there.